### ISSUE TITLE
QT UI Crashes when launching from history on a fresh Linux-x64 install

### ISSUE THREAD

**Issue by retro-wertz** [libretro/RetroArch#9030]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

### Expected behavior

crashless.

UPDATE:
looks like a **menu widget** issue. (running in window mode in x2 size if that matters) turning off menu widget does not crash regardless of how many times i redo the steps.

- can also happen when retroachievements are enabled. this causes RA to freeze.

### Actual behavior

as stated in title, using QT UI to launch from history causes retroarch to crash.

### Steps to reproduce the bug

Preparation: QTUI has to be enabled at startup. Items should already be in history (I was running nes games for this test). Crash happens on a fresh launch of Retroarch and running directly from the History.

* Run retroarch (has to be done from a fresh retroarch launch, with QTUI to run at startup)
* Run from the History playlist. this should crash immediately most of the time.

NOTE: This is done on a fresh install of MXLinux. RA compiles fine and QT works fine untill the said procedufe above that crash. (I normally do not use Debian but i just happen to be using atm.)

### Bisect Results
```
[INFO] [GLX]: glXSwapIntervalEXT(1)

Thread 1 "retroarch" received signal SIGSEGV, Segmentation fault.
0x00007ffff79a7c70 in QMetaCallEvent::~QMetaCallEvent() ()
   from /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
(gdb) bt full
#0  0x00007ffff79a7c70 in QMetaCallEvent::~QMetaCallEvent() ()
    at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#1  0x00007ffff79a7d09 in QMetaCallEvent::~QMetaCallEvent() ()
    at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#2  0x00007ffff79814d7 in QCoreApplicationPrivate::sendPostedEvents(QObject*, int, QThreadData*) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#3  0x00007ffff79d2dc3 in  () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#4  0x00007fffeedcdcd7 in g_main_context_dispatch ()
    at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#5  0x00007fffeedcdf48 in  () at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#6  0x00007fffeedcdffc in g_main_context_iteration ()
    at /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0
#7  0x00007ffff79d31cf in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () at /usr/lib/x86_64-linux-gnu/libQt5Core.so.5
#8  0x0000555555701709 in ui_application_qt_run(void*) ()
#9  0x00005555555e7d0e in rarch_main ()
#10 0x00007fffeff702e1 in __libc_start_main () at /lib/x86_64-linux-gnu/libc.so.6
#11 0x00005555555e4d0a in _start ()
(gdb) 

```

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: Linux mxlinux 4.19.0-5-amd64 #1 SMP Debian 4.19.37-2~mx17+1 (2019-05-15) x86_64 GNU/Linux
- Compiler: gcc version 6.3.0 20170516 (Debian 6.3.0-18+deb9u1)

**Comment by bparker06** [comment]
Can you turn on ASAN as well and see what it says after the crash?
Example for those unaware how to do that:
```
make clean
make -j$(nproc) DEBUG=1 SANITIZER=address,undefined
```

**Comment by retro-wertz** [comment]
heres a full log with asan https://hastebin.com/afafoqotew.sql

**Comment by twinaphex** [comment]
Can you retry tested against current master?

**Comment by retro-wertz** [comment]
thanks. 
i tested with latest commit and sadly is is the same issue for me.
i moved back to manjaro-xfce and its the same problem
https://hastebin.com/bidejadaje.shell
https://hastebin.com/visozivofe.shell
https://hastebin.com/oropelaqet.coffeescript

**Comment by bparker06** [comment]
Is this still an issue?

**Comment by retro-wertz** [comment]
yes, with menu widgets enabled.

**Comment by retro-wertz** [comment]
let me rephrase my last comment.. i cannot confirm if the exisiting issue is still happening because the issue relies that the qt ui has to start at launch,and currently this seems to be broken coz at start it just tries to open a window(see a quick opening and closing) and just closes so you only would see main RetroArch itself.

**Comment by retro-wertz** [comment]
both issue is still a problem for me, but i heard it was "working as intended" as official word. so im closing this.

feel free to reopen or request to reopen if anyone else can confirm these behaviors.
